R/sigopt_api_wrapper.R

```diff
@@ -1,3 +1,4 @@
+SIGOPT_TEST_API_KEY="UUEIBDHJUTKWOUAWSKBWFGLQSFYLQNJKZXAJVVSYNOQITYVK"
```

Comment:
I dont think we should embed this in here directly? can't we just include this in the test script?